### ISSUE TITLE
Can't exclude Android Auto from VPN

### ISSUE THREAD

**Issue by bcsti** [GrapheneOS/os-issue-tracker#2997]
The official Wireguard App for example does not show the Android Auto App provided by GrapheneOS in the list to exclude apps from the VPN connection.

The official Android Auto App however is available in the list.

Therefore I cannot use the VPN and have to turn it off every time I'm starting my car.

**Comment by thestinger** [comment]
The app we provide is the official app. That doesn't make sense.

**Comment by thestinger** [comment]
The app we provide is the official app... where are you obtaining whatever else you're installing?

**Comment by thestinger** [comment]
It's currently explicitly hidden from the Play Store.

**Comment by thestinger** [comment]
cc @muhomorr

**Comment by bcsti** [comment]
GOS:
![image](https://github.com/GrapheneOS/os-issue-tracker/assets/92046752/194883ec-8c88-4f60-85ab-17cff867fb53)

Stock:
![image](https://github.com/GrapheneOS/os-issue-tracker/assets/92046752/726895d8-2a89-44eb-8176-99e233981e5c)

**Comment by muhomorr** [comment]
I'm unable to reproduce this issue with Wireguard version 1.0.20231018

**Comment by thestinger** [comment]
@bcsti Is it installed in the same profile? VPNs are per-profile.

**Comment by bcsti** [comment]
Yes, all my VPNs and gapps are installed in the owner profile which is the only profile. 
Wireguard is installed via the Play Store.
I have also rebooted my phone several times.

The AA app is still not shown in both of my VPN Apps.
On my other stock phones the AA App is aviable to be excluded in the same VPN Apps. 
![Screenshot_20240103-180231](https://github.com/GrapheneOS/os-issue-tracker/assets/92046752/51674ade-d3a2-471d-b547-89a7283db832)
![Screenshot_20240103-180032](https://github.com/GrapheneOS/os-issue-tracker/assets/92046752/a0419bf2-182c-4306-a7b0-85170571536a)

**Comment by muhomorr** [comment]
Check whether Android Auto has the Network permission. Wireguard doesn't show apps that don't have Network permission in that list.

**Comment by bcsti** [comment]
Yes, you are right. After enabling Network access, AA is shown in Wireguard now. Thanks!
